assets/src/edit-story/elements/text/edit.js

```diff
@@ -54,7 +54,18 @@ import { generateParagraphTextStyle, getHighlightLineheight } from './util';
 // on the content and properties.
 const Wrapper = styled.div`
   ${elementFillContent}
-  ${elementWithBackgroundColor}
+
+  &::after {
+    content: '';
+    display: block;
+    position: absolute;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    border: 1px solid ${({ theme }) => theme.colors.mg.v1}70;
```

Comment:
So this works as long as the color in the theme is a 6-digit hex, but if someone changes it to a 3-digit hex or any other color definition (`black`, `rgb()`, etc), this breaks. I think it's better to use [the `rgba()` util](https://polished.js.org/docs/#rgba) from `polished` here.

Reply:
Good catch, although I'm not sure what happened here, we had this typo in the past but it was removed 🤔    https://github.com/google/web-stories-wp/pull/524#discussion_r390571722